MyLinkedList/LinkedListClass.cs

```diff
@@ -1,8 +1,764 @@
+            {
+                Node<T> current = _head;
+                T dataMax = _head.Data;
+
+                for (int i = 1; i < Count; i++)
+                {
+                    if (dataMax.CompareTo(current.Next.Data) == -1)
+                    {
+                        dataMax = current.Next.Data;
+                    }
+                    current = current.Next;
+                }
+
+                return dataMax;
+            }
+
+            throw new InvalidOperationException();
+        }
+
+        public int GetMinIndex()
+        {
+            if (!(_head is null))
+            {
+                Node<T> current = _head;
+                T dataMin = _head.Data;
+                int index = 0;
+
+                for (int i = 1; i < Count; i++)
+                {
+                    if (dataMin.CompareTo(current.Next.Data) == 1)
+                    {
+                        index = i;
+                        dataMin = current.Next.Data;
+                    }
+                    current = current.Next;
+                }
+
+                return index;
+            }
+
+            throw new InvalidOperationException();
+        }
+
+        public T GetMin()
+        {
+            if (!(_head is null))
+            {
+                Node<T> current = _head;
+                T dataMin = _head.Data;
+
+                for (int i = 1; i < Count; i++)
+                {
+                    if (dataMin.CompareTo(current.Next.Data) == 1)
+                    {
+                        dataMin = current.Next.Data;
+                    }
+                    current = current.Next;
+                }
+
+                return dataMin;
+            }
+
+            throw new InvalidOperationException();
+        }
+
+        public void Sort(bool isAscending = true)
+        {
+            Node<T> sorted = null;
+            Node<T> current = _head;
+            int coefficient = isAscending ? -1 : 1;
+
+            while (current != null)
+            {
+                Node<T> next = current.Next;
+
+                sorted = SortedInsert(current, sorted, coefficient);
+                current = next;
+            }
+
+            _head = sorted;
+        }
+
+        public void Reverse()
+        {
+            if (!(_head is null))
+            {
+                Node<T> current = _head;
+                Node<T> previos = null;
+                Node<T> next;
+
+                do
+                {
+                    next = current.Next;
+                    current.Next = previos;
+                    previos = current;
+                    current = next;
+                }
+                while (!(current is null));
+
+                _head = previos;
+            }
+        }
+
+        public void HalfReverse()
+        {
+            if (!(_head is null))
+            {
+                Node<T> current = _head;
+                Node<T> temp = _head;
+
+                if (Count % 2 == 0)
```

Comment:
You could use % 2 as operator to get implisit `1` in your code to get only one block of code instead of `if\else`